LoginPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import { useHistory } from 'react-router-dom';
+
+const LoginSchema = Yup.object().shape({
+  email: Yup.string().email('Invalid email').required('Required'),
+  password: Yup.string().min(8, 'Too Short!').max(50, 'Too Long!').required('Required'),
+});
+
+const LoginPage = () => {
+  let history = useHistory();
+
+  return (
+    <div className="login-page">
+      <div className="image-section">
+        {/* Your image here */}
+      </div>
+      <div className="form-section">
+        <h1>Login</h1>
+        <p>Enter your email and password to access your account</p>
+        <Formik
```

Comment:
Consider abstracting the form handling logic into a separate custom hook. This will improve the modularity and maintainability of your code.